cli.php

```diff
@@ -11,6 +11,10 @@ class WPSuperCache_Command extends WP_CLI_Command {
 	 * @synopsis [--post_id=<post-id>] [--permalink=<permalink>]
 	 */
 	function flush( $args = array(), $assoc_args = array() ) {
+		require_once( WPCACHEHOME . '/wp-cache-phase1.php' );
+		global $WPSC_HTTP_HOST;
+		$WPSC_HTTP_HOST = parse_url(get_site_url())['host'];
```

Comment:
@danielbachhuber Thanks, will use `home_url()` then. On a related note, just wondering why it makes sense to explicitly define `url` and `path` in `wp-cli.yml`. Seems redundant.